base/src/META-INF/blaze-base.xml

```diff
@@ -409,7 +409,9 @@
     <registryKey defaultValue="true"
                  description="By default Bazel plugin takes control and opens project if .ijwb folder is present. This property can be used to disable this behavior to allow to open .idea or other project models if they are exists."
                  key="bazel.project.auto.open.if.present"/>
-
+    <registryKey defaultValue="false"
```

Comment:
is the plan to make this the default behaviour?

Reply:
Nope, but I prefer to keep this option in case the users would like to debug or analyze performance of the query. Previously they could copy it from the cmd line. Now it's in a file disappearing right after the query is executed.

Reply:
I see, I think we need to collect these registry keys and make them visible in the plugin documentation

Reply:
Yeah, do you think it would be possible to parse these xml files to retrieve names and the descriptions, and then upload them to ij.bazel.build on regular basis?